src/datatypes/numeric.js

```diff
@@ -24,7 +24,14 @@ function generateFunctions(bufferReader,bufferWriter,size)
     };
   };
   var writer=function(value, buffer, offset) {
-    buffer[bufferWriter](value, offset);
+    try {
```

Comment:
hmm that's pretty useful for debugging precisely, gives more context than when the error is eventually caught, way later.